test/entry.ts

```diff
@@ -37,7 +39,7 @@ const structToObj = (struct: {}) => {
 };
 
 describe('Entry', () => {
-  // tslint:disable-next-line no-any variable-name
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
what happened to the `variable-name` disable for the cap `E` in `Entry` within a `let`. do the linting rules no longer have this check?

Reply:
`linter` does not complain about the variable name, just checked the `gts` package, it might be related with this rule: `"@typescript-eslint/camelcase": "off",`?

Reply:
`variable-name` is now `@typescript-eslint/naming-convention`, see https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md#style

I've opened https://github.com/google/gts/issues/486